client/src/pages/SplashScreen/splashScreenStyles.ts

```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    minHeight: '100vh',
+    alignItems: 'center',
+    alignContent: 'center',
+  },
+  input: {
+    height: '3rem',
+    padding: 5,
+    background: 'white',
+    textAlign: 'center',
+    borderRadius: 5,
+    boxShadow: '1px 1px 4px rgb(0,0,0,.3)',
```

Comment:
Really minor, but I think we should be consistent with style across all our textfields and inputs in the app. The mocks show both styles (login screen vs add card dialog) so I think we could pick whichever style we prefer

Reply:
You're right. Thanks!